WcaOnRails/app/views/doorkeeper/authorized_applications/index.html.erb

```diff
@@ -1,5 +1,16 @@
-<header class="page-header">
-  <h1><%= t('doorkeeper.authorized_applications.index.title') %></h1>
+<% provide(:title, t('doorkeeper.authorized_applications.index.title')) %>
+<%
+  # FIXME panic button, I removed the navbar.
+  # For some reason the locale is correctly set in the params,
+  # but the default_url_options used in the navigation doesn't have it...
```

Comment:
I have flashbacks of a similar issue where the locale depended on the load order of some 3rd party gems. I never resolved it. :scream: